source/exportdialog.cpp

```diff
@@ -18,9 +18,15 @@ ExportDialog::~ExportDialog()
     delete ui;
 }
 
-void ExportDialog::setCel(D1CelBase *c)
+void ExportDialog::initialize(QJsonObject *cfg, D1CelBase *c, D1Min *m, D1Til *t, D1Sol *s, D1Amp *a)
 {
+    this->configuration = cfg;
+
     this->cel = c;
+    this->min = m;
+    this->til = t;
+    this->sol = s;
+    this->amp = a;
```

Comment:
sol and amp are now unused in this file and can be removed

Reply:
It seems `exportLevelSubtiles` checks if `sol` is valid. Since it is not really performance critical I think it is better to leave it for the moment, because the code is very much in the expanding phase.